### PR TITLE
Adds lazarus kit, adds a explorer webbing to crusher kit, also mini pickaxes!

### PR DESCRIPTION
**Pull request by improvedname**
**What does this PR do:**
This pr adds a new based kit thought by me which consists out of.
1 lazarus belt, 1 capsule, 1 injector, and 1 pack of truama/burn treatment packs to heal your lazarus mob with. gotta catch them all and fight with your mobs against other lavaland creatures! 🗡 

Also adds a explorer webbing to crusher kit so inventory management is more sane as you lose the backpack slot with the crusher. (also gives you a belt space to store your mini extinguisher

**Changelog:**
:cl: Improvedname
add: Adds lazarus kit as a mining voucher startup
add:ports compact pickaxes from TG
tweak: Adds a explorer webbing to crusher kit
/:cl:

### CODE REVIEW

File: code/modules/mining/equipment_locker.dm [hunk 1]
```diff
@@ -597,7 +597,7 @@
 	..()
 
 /obj/machinery/mineral/equipment_vendor/proc/RedeemVoucher(obj/item/mining_voucher/voucher, mob/redeemer)
-	var/items = list("Kinetic Accelerator", "Survival Capsule and Explorer's Webbing", "Resonator Kit", "Minebot Kit", "Crusher Kit", "Mining Conscription Kit", "Advanced Scanner")
+	var/items = list("Lazarus Kit", "Kinetic Accelerator", "Survival Capsule and Explorer's Webbing", "Resonator Kit", "Minebot Kit", "Crusher Kit", "Mining Conscription Kit", "Advanced Scanner")
```
Comment (Markolie) [inline code/modules/mining/equipment_locker.dm]: This shouldn't really go as the first item on the list, considering it's less important than the other ones. I'd put it at the end.

Comment (improvedname) [reply code/modules/mining/equipment_locker.dm]: oh right its the click and enter meme. eh but sure

**Comment by Fox-McCloud** [comment]
The Lazarus kit seems ok, but adding a belt to the crusher is OP as heck and shouldn't be included.

You can spend an entire voucher for just a mining belt........or spend it to get a mining belt+crusher+extinguisher.

**Comment by variableundefined** [comment]
As above, please remove belt from the crusher

**Comment by Fox-McCloud** [comment]
Closing this. No response; if you want to remake the PR with only the lazarus kit, that'd be fine.

**Pull request closed without merging**